### ISSUE TITLE
parse.find: TypeError: Property 'next' of object [object Object] is not a function

### ISSUE THREAD

**Issue by davious** [rc0x03/node-osmosis#14]
When I use parse and try to find,
```
osmosis.parse('<html><body>text</body></html>').find("body");
```

I get the following error
```
node-osmosis/lib/promise.js:235
                this.next(parser.parse(data));
                     ^
TypeError: Property 'next' of object [object Object] is not a function
    at null.initialized (node-osmosis/lib/promise.js:235:8)
    at null.find (node-osmosis/lib/promise.js:82:9)
```

**Comment by rc0x03** [comment]
This issue has been fixed in 0.0.6

**Comment by davious** [comment]
:+1:
